src/web/lib/hakuneko/frontend@classic-dark/pages.html

```diff
@@ -516,6 +525,42 @@
                     this.$.container.scrollTop = previousOffset * this.$.container.scrollHeight / previousHeight;
                 //}.bind( this ), 0 );
             }
+
+            /**
+             * Dynamically change the scrolling to stop at the end of images or skip to the start of the next image
+             * Usefull for manga reading but ugly with webtoons
+             */
+            scrollMagic(defaultDistance) {
+                let images = this.$.container.getElementsByClassName('image');
```

Comment:
May use `container.querySelectorAll('image')`

Reply:
Indeed, easier to expand if needed.